.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - crystal_version: nightly
             experimental: true
     runs-on: ubuntu-latest
-    container: crystallang/crystal:${{ matrix.crystal_version }}-alpine
+    container: crystallang/crystal:${{ matrix.crystal_version }}
```

Comment:
This is because Ameba tanks on 1.6.0 under alpine

ref: https://github.com/crystal-ameba/ameba/issues/354